bats/tests/helpers/profile.bash

```diff
@@ -137,17 +137,19 @@ export_profile() {
     fi
 }
 
-# Add boolean setting; value must be "true" or "false"
+# Set a profile setting to a boolean; value must be "true" or "false"
+# The profile must exist before calling this function.
 add_profile_bool() {
     local setting=$1
     local value=$2
 
+    assert profile_exists
```

Comment:
If you add `|| return` to other statements later in this function, then you should add it to the `assert` as well.